apps/web-giddh/src/app/downloads/components/imports/imports.component.ts

```
@@ -353,38 +356,15 @@ export class ImportsComponent implements OnInit, OnDestroy {
-                                    };
-                                </script>
-                            </body>
-                        </html>
-                    `);
-                win.document.close();
-            };
-            reader.readAsDataURL(blob);
+    public downloadFile(url: any): void {
+        if (url) {
+            let fileName = url.substring(url.lastIndexOf('/') + 1);
+            return download(fileName, url, "");
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)